client/src/containers/CurrentEvent/CurrentEvent.js

```diff
@@ -19,20 +17,14 @@ import { fetchCurrentEvent } from '../../actions';
 import ContentHeader from '../../components/ContentHeader/ContentHeader';
 import moment from 'moment';
 import DescriptionContainer from '../../components/DescriptionContainer/DescriptionContainer';
-import EditUserProfile from '../EditProfile/EditUserProfile';
 import { Link } from 'react-router-dom';
 import { AttendeeType } from '../../types/attendee-types';
 import './CurrentEvent.css';
 
 const DATE_FORMAT = 'LLLL';
 
-class CurrentEvent extends Component {
+class CurrentEvent extends Component<Props> {
   eventId: string;
```

Comment:
@taleymirza I don't think we need to define eventId type here?

Reply:
@oahmedqureshi I am not working on this PR, @JavaEria! please review